client/me/privacy/main.jsx

```diff
@@ -84,7 +84,7 @@ const Privacy = createReactClass( {
 							<p>{ translate( 'We are committed to your privacy and security.' ) }</p>
 							<p>
 								{ translate(
-									'This information helps us improve our products, make marketing to you more ' +
+									'The information you choose to share helps us improve our products, make marketing to you more ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'This information helps us improve our products, make marketing to you more relevant, personalize your WordPress.com experience, and more as detailed in our {{privacyPolicyLink}}privacy policy{{/privacyPolicyLink}}.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).